server/commands/write_authzmodel.go

```diff
@@ -55,7 +55,7 @@ func (w *WriteAuthorizationModelCommand) Execute(ctx context.Context, req *openf
 	}
 
 	utils.LogDBStats(ctx, w.logger, "WriteAuthzModel", 0, 1)
-	if err := w.backend.WriteAuthorizationModel(ctx, req.GetStoreId(), id, typeSystem.GetTypeDefinitions()); err != nil {
```

Comment:
Can we just change this signature to write the AuthorizationModel itself? That'd be quite a bit nicer.

```
func ... WriteAuthorizationModel(ctx context.Context, model *openfgapb.AuthorizationModel) error {
    ...
}
```

Reply:
I thought about that, but we lose the typing on `SchemaVersion` if we do that as the schema version in openfgapb.AuthorizationModel is a string.

Reply:
Take the `*openfgapb.AuthorizationModel` and convert to the underlying serialized type (the enum) inside of the storage layer. I believe this is a decent approach since we're storing a string in a enum specific way within the storage layer.

Reply:
I like this idea and I am all for it, but there are millions of changes due to tests. I can do this now, or in a focused follow-up PR. WDYT?

Reply:
refactor later ™️